core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -553,6 +553,13 @@ public String toString() {
       .setDescription("The access key of GCS bucket.").build();
   public static final PropertyKey GCS_SECRET_KEY = new Builder(Name.GCS_SECRET_KEY)
       .setDescription("The secret key of GCS bucket.").build();
+  public static final PropertyKey OBS_ACCESS_KEY = new Builder(Name.OBS_ACCESS_KEY)
+      .setDescription("The access key of OBS bucket.").build();
+  public static final PropertyKey OBS_ENDPOINT = new Builder(Name.OBS_ENDPOINT)
+      .setDefaultValue("obs.myhwclouds.com")
```

Comment:
do we want a fake default value here?
we can skip setting default value (=null)

Reply:
it's a meaningful default value according to huawei cloud's doc.

Reply:
sg